Formula/kim-api-v2.rb

```diff
@@ -0,0 +1,33 @@
+class KimApiV2 < Formula
+  desc "The Knowledgebase of Interatomic Models (KIM) API"
+  homepage "https://openkim.org"
+  url "https://s3.openkim.org/kim-api/kim-api-v2-2.0.0.txz"
+  sha256 "beb9eb47cfe9ac9a4ad22528da89a67415a5b033075a9318a5d4a800dd4a3f3d"
+
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "gcc" # for gfortran
+
+  def install
+    args = std_cmake_args
+    args << "-DCMAKE_INSTALL_PREFIX=/usr/local"
```

Comment:
`/usr/local` should be `HOMEBREW_PREFIX`

Reply:
right